system-int-test/test/cmr/system_int_test/admin/cache_api_test.clj

```diff
@@ -152,27 +156,27 @@
            (is (= ["You do not have permission to perform that action."] errors))))))
 
     (testing "list cache keys"
-      (are [url cache cache-keys]
-           (let [response (list-cache-keys url cache admin-read-token)]
-             (is (= (set cache-keys) (set response))))
-
-        (url/indexer-read-caches-url) "acls" ["acls"]
-        (url/indexer-read-caches-url) "indexer-index-set-cache" ["concept-indices" "concept-mapping-types"]
-        (url/indexer-read-caches-url) "token-imp" [["ABC-2" "read"] ["ABC-1" "read"]]
-        (url/mdb-read-caches-url) "token-imp" [["mock-echo-system-token" "update"]
-                                               ["ABC-1" "read"]
-                                               ["ABC-2" "read"]]
-        (url/ingest-read-caches-url) "token-imp" [[nil "update"]
-                                                  ["ABC-1" "read"]
-                                                  ["ABC-2" "read"]]
-        (url/search-read-caches-url) "acls" ["acls"]
-        (url/search-read-caches-url) "collections-for-gran-acls" []
-        (url/search-read-caches-url) "has-granules-map" []
-        (url/search-read-caches-url) "index-names" [":acls-hash-code" ":collection-granule-aggregation-cache-hash-code" ":concept-indices-hash-code" ":concept-mapping-types-hash-code" ":providers-hash-code" ":write-enabled-hash-code" "collection-granule-aggregation-cache" "kms" "humanizer-cache"]
-        (url/search-read-caches-url) "token-imp" [["ABC-1" "read"] ["ABC-2" "read"]]
-        (url/search-read-caches-url) "token-sid" ["ABC-2" "ABC-1"]
-        (url/search-read-caches-url) "xsl-transformer-templates" []
-        (url/search-read-caches-url) "token-user-id" ["ABC-1" "ABC-2"])
+      (are3 [url cache cache-keys]
```

Comment:
nice job converting the are to are3

Reply:
I love answering the call, and it gives me a chance to try to understand the test